### PR TITLE
[WebAPI] Make the test suites support Ostro IoT platform

### PR DESCRIPTION
**Pull request by qiuzhong**
Make the webapi test suites support Ostro IoT platform.

BUG=https://crosswalk-project.org/jira/browse/CTS-1886

### CODE REVIEW

File: tools/build/build_iot.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2015 Intel Corporation.
```
Comment (Honry) [inline tools/build/build_iot.py]: Please use 2016 copyright.

File: tools/build/build_iot.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+import os
+import glob
+import json
+import shutil
+import utils
+import varshop
+
+
+def regenIoTManifest(manifest_path, index_file = 'index.html'):
```
Comment (Honry) [inline tools/build/build_iot.py]: As we talked in the meeting, this function is unnecessary.

**Comment by qiuzhong** [comment]
@Honry , updated.

File: tools/build/build_iot.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            manifest['start_url'] = index_file
+
+            json.dump(manifest, fp, indent = 4)
+    except Exception:
+        return False
+
+    return True
+
+
+def packIOT(build_json = None, app_src = None, app_dest = None, app_name = None):
```
Comment (Honry) [inline tools/build/build_iot.py]: typo, "IoT"

**Comment by Honry** [comment]
Thanks, LGTM

**Pull request merged**